.github/workflows/ci.yml

```diff
@@ -47,15 +47,6 @@ jobs:
         with:
           gem: "${{ matrix.gem }}"
           ruby: "3.0"
-      - name: "Test Ruby 2.7"
-        if: "${{ matrix.os == 'ubuntu-latest' }}"
-        uses: ./.github/actions/test_gem
-        with:
-          gem: "${{ matrix.gem }}"
-          ruby: "2.7"
-          yard: true
```

Comment:
You will want to move these to the 3.0 block otherwise it won't run rubocop, yardoc, or gem build

Reply:
https://github.com/open-telemetry/opentelemetry-ruby/pull/1459/commits/dd1b69a41c72b375b95c517f6f683c4d4d221783